cmd/main.go

```diff
@@ -2,12 +2,20 @@ package main
 
 import (
 	"github.com/CESARBR/knot-cloud-storage/internal/config"
+	"github.com/CESARBR/knot-cloud-storage/pkg/data"
 	"github.com/CESARBR/knot-cloud-storage/pkg/logging"
 	"github.com/CESARBR/knot-cloud-storage/pkg/server"
 )
 
+var dao = data.DataDAO{}
+
 func main() {
 	config := config.Load()
+
+	dao.Database = config.MongoDB.Name
```

Comment:
We need to use the `main` to instantiate the data components and inject them as interactors dependencies. Please, take a look at the +/- expected code:

```
database := data.NewMongoDB(config.MongoDB.Name, config.MongoDB.host)
dataStore := data.NewDataStore(database)

saveDataInteractor := interactor.NewSaveData(database)
...
```

Reply:
Done!